action.yml

```diff
@@ -138,7 +152,7 @@ runs:
         echo "CA_GH_ACTION_IMAGE_TO_BUILD=${CA_GH_ACTION_IMAGE_TO_BUILD}" >> $GITHUB_ENV
 
     - name: Determine name of image to build if not provided
-      if: ${{ inputs.imageToBuild == '' }}
+      if: ${{ inputs.acrName != '' && inputs.imageToBuild == '' }}
```

Comment:
Little confused by this and the task above. So in the case that `acrName` exists and `imageToBuild` does not, wouldn't we write `CA_GH_ACTION_IMAGE_TO_BUILD` to `$GITHUB_ENV` twice? Same with below, like with `CA_GH_ACTION_IMAGE_TO_DEPLOY`. Guess I'm just trying to figure out why we need to `Export name of image to deploy to environment variable` and then `Update name of image to deploy if not provided`

Reply:
Good point, there's no reason to assign it twice -- I think this was just a method to make sure the environment variable was assigned to _at least_ some value so that it could be used later, but I can add some simple if-statements to the first checks so that the value is only ever assigned once.